docs/start/latest-updates.mdx

```diff
@@ -76,7 +76,14 @@ On the IBM Quantum blog, we take a deep dive into the new beta release with a sp
 
 ## Qiskit IBM Runtime service updates
 
-This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use. 
+This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use.
+
+### 1 July 2024
+
+The following endpoints are deprecated and will be removed on or after 1 October 2024: `GET /stream/jobs` and `GET /stream/jobs/{id}`. This removal has the following impacts:
+
+- After the endpoints are removed, `job.stream_results()` and `job.interim_results()` will be removed from the [qiskit-ibm-runtime](/api/qiskit-ibm-runtime) client.
+- Additional methods, such as `job.result()` currently use the deprecated endpoints.  Upgrade to `qiskit_ibm_runtime` 0.25 or later before the endpoints are removed to avoid disruption.
```

Comment:
So we don't have qiskit-ibm-runtime 0.25 yet 😅. The 2 PRs needed for this were merged, so @kt474 can presumably make a release tomorrow. Do we want to wait for that?

Reply:
Sure, I can do a release tmrw

Reply:
That seems sensible